runtime/frequency-rococo/src/lib.rs

```diff
@@ -462,6 +462,73 @@ impl pallet_democracy::Config for Runtime {
 	type OperationalPreimageOrigin = pallet_collective::EnsureMember<AccountId, CouncilCollective>;
 }
 
+parameter_types! {
+	/// The time proposals are collected before payout
+	/// as well as when the burn of left over funds happens (if any)
+	/// Setup on a 1-week scheduled for Rococo
+	pub SpendPeriod: BlockNumber = prod_or_local_or_env!(7 * DAYS, 10 * MINUTES, "FRQCY_SPEND_PERIOD");
```

Comment:
since we separated we runtimes, it's a bit weird that we are using `prod_or_local_or_env`. It feels like maybe we can combine those runtime s and just use this on every param. It would be easier than duplicate everything in these 2 files

Reply:
can you please explain why we cant do this with `prod_or_fast`?

Reply:
@aramikm It is tempting to do a merge and use a macro like this. I worry that might end up really nasty as well however.

We only use it in `frequency-rococo` however as that's the "shared" one between instant, local rococo, and public rococo. Mainnet should never see this, or if it does, it would never do anything.

Reply:
@enddynayn `prod_or_fast` macro is what this is, just swapping out the feature flag it uses.